src/app/users/sites.ts

```diff
@@ -494,23 +496,23 @@ export function getSiteFromPath(req: { path: string }) {
 
 /*
 insert into shinuim.users
-( id, org, name, phone, adminNotes, createDate, createUserId, admin, dispatcher, trainee, manageDrivers, deleted, lastUpdateView, addressApiResult, address, okCategories)  
```

Comment:
My IDE appears to have optimized away various whitespaces in this file. I hope this doesn't break anything.